js/kubewarden/host_capabilities/oci/manifest_digest/manifest_digest.test.ts

```diff
@@ -0,0 +1,104 @@
+import { ManifestDigest } from "./manifest_digest";
+import * as HostCall from "../../index";
+
+class MockWasmHost {
+    private knownImages: Map<string, string> = new Map();
+
+    constructor(imageDigests: Record<string, string> = {}) {
```

Comment:
I'm not a JS developer, it might be a stupid question.

Can't we use the same type both for `knownImages `and for `imageDigests` and then just copy the object instance?